guides/keras_cv/cut_mix_mix_up_and_rand_augment.py

```diff
@@ -0,0 +1,398 @@
+"""
+Title: CutMix, MixUp, and RandAugment image augmentation with KerasCV
+Author: [lukewood](https://lukewood.xyz)
+Date created: 2022/04/08
+Last modified: 2022/04/08
+Description: Use KerasCV to augment images with CutMix, MixUp, RandAugment, and more!
+"""
+
+"""
+## Overview
+
+KerasCV makes it easy to assemble state of the art, industry grade data augmentation
+pipelines for image classification and object detection tasks.  KerasCV offers a wide
+suite of preprocessing layers implementing common data augmentation techniques.
+
+Perhaps three of the most useful layers are `CutMix`, `MixUp`, and `RandAugment`.  These
+layers are used in nearly all state of the art image classification pipelines.
+
+This guide will show you how to compose these layers into your own state of the art data
+augmentation pipeline for image classification tasks.  This guide will also walk you
+through the process of customizing a KerasCV data augmentation pipeline.
+"""
+
+"""
+## Imports and Setup
+
+This tutorial requires you to have KerasCV installed:
+
+```shell
+!pip install keras-cv
+```
+
+We will also begin by import all required packages:
+"""
+
+import keras_cv
+import matplotlib.pyplot as plt
+import tensorflow as tf
+import tensorflow_datasets as tfds
+from tensorflow.keras import applications
+from tensorflow.keras import losses
+from tensorflow.keras import optimizers
+
+"""
+## Data Loading
+
+This guide uses the oxford_flowers102 dataset for demonstration purposes.
```

Comment:
Maybe use the original name of the dataset with an official link?

Reply:
Done